R/DatasetSummaryResponse.R

```diff
@@ -0,0 +1,54 @@
+'data.world-r
+Copyright 2017 data.world, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License.
+
+This product includes software developed at data.world, Inc.(http://www.data.world/).'
+
+#' DatasetSummaryResponse per https://api.data.world/v0/swagger.json
```

Comment:
Not sure , we can consider this to be "internal" considering we do expose the definition of that POJO in https://api.data.world/v0/swagger.json . This class is more or less a translation of that definition in R